staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/integrator.go

```diff
@@ -63,7 +66,18 @@ type integrator struct {
 
 // NewIntegrator makes one that uses the given clock
 func NewIntegrator(clock clock.PassiveClock) Integrator {
+	ans := &integrator{
+		clock:    clock,
+		lastTime: clock.Now(),
+	}
+	ans.name = fmt.Sprintf("%p", ans)
+	return ans
+}
```

Comment:
BTW - it seems that other than some tests, the integrator isn't used anywhere in the code now.
Did we just forget to remove it when you were refactoring the metrics couple months ago?

Reply:
I left it because it was useful for the tests and seemed like something that might turn out to have more uses in the future.  This PR shows that was right; this PR puts Integrator to use in non-test code.